i18n/en/form.json

```diff
@@ -32,7 +32,10 @@
     },
     "dueDate": {
       "name": "Due Date",
-      "placeHolder": "Select a Date"
+      "placeHolder": "Select a custom date",
+      "addSomeDaysToCurrentDate": "Add some days to the current date",
+      "addDaysToCurrentDate" : " - Payment seven days after invoice date",
```

Comment:
This actually doesn't make sense

Reply:
Do you mean line number 37? I thought, I've updated this string to something meaningful.

Reply:
Yes, on line 37, the key and the value don't match.